src/test/java/seedu/address/logic/commands/FavouriteCommandTest.java

```diff
@@ -28,6 +29,8 @@
     private final Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs(), null);
     private final Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs(), null);
 
+    private Index targetIndex;
+
```

Comment:
Did you add this by mistake? It does not seem to be used below.

Reply:
Fixed it.